boot/freeldr/fdebug/lang/ro-RO.rc

```diff
@@ -50,36 +56,36 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Opțiuni de conectare"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Alegeți portul COM (ex. COM1):", IDC_STATIC, 7, 7, 108, 8
+    LTEXT "Treceți portul COM (ex. COM1):", IDC_STATIC, 7, 7, 108, 8
     EDITTEXT IDC_COMPORT, 7, 17, 182, 14, ES_AUTOHSCROLL
-    LTEXT "Rata baud (ex. 115200):", IDC_STATIC, 7, 38, 114, 8
+    LTEXT "Treceți rata baud (ex. 115200):", IDC_STATIC, 7, 38, 114, 8
     EDITTEXT IDC_BAUTRATE, 7, 48, 182, 14, ES_AUTOHSCROLL
-    DEFPUSHBUTTON "Con&firmă", IDOK, 45, 79, 50, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 100, 79, 50, 14
+    DEFPUSHBUTTON "OK", IDOK, 45, 79, 50, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 100, 79, 50, 14
 END
 
 IDD_CAPTURE DIALOGEX 0, 0, 251, 95
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Fișierul de captură"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Numele fișierului:", IDC_STATIC, 7, 17, 62, 8
+    LTEXT "Nume fișier capt.:", IDC_STATIC, 7, 17, 62, 8
```

Comment:
The string wouldn't full fit in the static label?

Reply:
No